### PR TITLE
User management

### PR DESCRIPTION
**Pull request by dhganey**
This adds a signup screen to create user accounts. No verification is required for the account, so anyone will be able to make one if they have access to the URL. We should consider ways to validate users.

Additionally, this adds a simple user management page. Users can view each other's information and sets. Admin users (a boolean in the database) have the option to delete users on this page.

### CODE REVIEW

**Comment by jrnorth** [comment]
One thing I can see is that there's an extra column on the RHS of the table on the users page if you're not an admin.

File: eorlive_prototype/app/templates/signup.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      <dd><input type='text' name='lname' required>
+      <dt>Email address:
+      <dd><input type='text' name='email' required>
+      <dt>Username:
+      <dd><input type='text' name='username' required>
+      <dt>Password:
+      <dd><input type='password' name='password' required>
+      <dt>Confirm Password:
+      <dd><input type='password' name='password2' required>
+      <dd><input type='submit' value='Log in'>
```
Comment (jrnorth) [inline eorlive_prototype/app/templates/signup.html]: This should be 'Sign up' instead of 'Log in'.

**Comment by dhganey** [comment]
@jrnorth damn, I thought I'd removed that extra column. It's supposed to just be 2 columns if not admin, I'll take another look at it tomorrow.

**Comment by dhganey** [comment]
Anyone object to this merging?

**Pull request merged**